geniza/corpus/migrations/0026_delete_documentprefetchableproxy.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.8 on 2022-01-20 21:16
+
+from django.db import migrations
+
+
+def reassign_log_entries(apps, schema_editor):
+    LogEntry = apps.get_model("admin", "LogEntry")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    document_type = ContentType.objects.get(app_label="corpus", model="document")
+    for log_entry in LogEntry.objects.filter(
+        content_type__model="documentprefetchableproxy"
+    ):
+        log_entry.content_type = document_type
+        log_entry.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("admin", "0003_logentry_add_action_flag_choices"),
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("corpus", "0025_documentprefetchableproxy"),
+    ]
+
+    operations = [
+        # delete DocumentPrefetchableProxy model
+        migrations.DeleteModel(
+            name="DocumentPrefetchableProxy",
+        ),
+        # reassign all log entries associated with DocumentPrefetchableProxy
+        # instances to Documents instead
+        migrations.RunPython(reassign_log_entries, migrations.RunPython.noop),
```

Comment:
does this order work fine? the prefetchable proxy content type isn't deleted when the model is deleted, right?

Reply:
That's right, but it doesn't hurt to change the order and it makes more logical sense the other way.

Reply:
Done in 92e4487 👍